source/index.html.md

```diff
@@ -475,16 +475,16 @@ This endpoint updates a PaymentSchedule record
 > **Example PaymentSchedule body**
 
 ```json
-  {
-    "amount": 74,
-    "graceDate": "2020-03-13",
-    "id": "6bb42d33-c2a2-4763-90e4-ff1702fc9951",
-    "reminderDate": "2020-03-30",
-    "scheduledFor": "2020-04-01",
-    "shortenedUrl": "https://rvkn.app/3330eb10",
-    "status": "AVAILABLE",
-    "subscriptionId": "3330eb10-8a8c-11ec-a8a3-0242ac120002"
-  }
+{
```

Comment:
The only fields relevant when updating a payment schedule is `id`, `amount` and `date`. So revise the fields accordingly.